airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json

```
@@ -57,17 +57,17 @@
     "ad_group_criterion.keyword.match_type": {
       "type": ["null", "string"],
       "enum": ["BROAD", "EXACT", "PHRASE", "UNKNOWN", "UNSPECIFIED"]
-    },
-    "metrics.historical_quality_score": {
-      "type": ["null", "integer"]
-    },
+    }
```

@LuliRib it is missing a comman and breaking the tests.
```suggestion
    },
```